docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -280,6 +281,15 @@ attempts to obtain a node to be returned. See [Selector Timeout](../test-api/sel
 testcafe ie my-tests --selector-timeout 500000
 ```
 
+### --assertion-timeout \<ms\>
+
+Specifies the amount of time, in milliseconds, within which [assertion](../test-api/assertions/index.md) make
```

Comment:
+Specifies the amount of time, in milliseconds, within which TestCafe tries to successfully execute an [assertion](../test-api/assertions/index.md).